### PR TITLE
Add PaymentFixture

### PR DESCRIPTION
**Pull request by TheMilek**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13                   |
| Bug fix?        | no                                                      |
| New feature?    | yes                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no  |
| License         | MIT                                                          |

This fixture allows completeing percentage amount of orders 💃🏽

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentFixture.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    private OptionsResolver $optionsResolver;
+
+    public function __construct(
+        PaymentRepositoryInterface $paymentRepository,
+        StateMachineFactoryInterface $stateMachineFactory,
+        ObjectManager $paymentManager,
+    ) {
+        $this->paymentRepository = $paymentRepository;
+        $this->stateMachineFactory = $stateMachineFactory;
+        $this->paymentManager = $paymentManager;
```
Comment (coldic3) [inline src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentFixture.php]:
```suggestion
    private Generator $faker;

    private OptionsResolver $optionsResolver;

    public function __construct(
        private PaymentRepositoryInterface $paymentRepository,
        private StateMachineFactoryInterface $stateMachineFactory,
        private ObjectManager $paymentManager,
    ) {
```

**Comment by coldic3** [comment]
I would be for adding it to `custom_fixtures.yaml`, eg.:
```yaml
payments:
    options:
        percentage_completed: 75
```

**Comment by TheMilek** [comment]
> I would be for adding it to `custom_fixtures.yaml`, eg.:
> 
> ```yaml
> payments:
>     options:
>         percentage_completed: 75
> ```

Good point! Handled now 😅

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Kamil! :tada:

**Pull request merged**